Spotted something suspicious

JoshuaJacobson opened the pull request:
This may not be the source of your issues, but it may be a contributing factor.

JoshuaJacobson committed:
Spotted something suspicious

Minisolarclown commented:
It's only something that I (programmer) use, so it never gets touched during the matches. I'm the only one that even knows about that button. I even forgot that was even there. Which of my Chief Delphi threads are you coming from?

JoshuaJacobson commented:
I'm from team 5086. CSA Alex Irwin sent me your code.

On Thu, Apr 5, 2018, 14:59 Connor Williams <notifications@github.com> wrote:

> It's only something that I (programmer) use, so it never gets touched
> during the matches. I'm the only one that even knows about that button. I
> even forgot that was even there. Which of my Chief Delphi threads are you
> coming from?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/CyberCoyotes/2018PowerUp/pull/1#issuecomment-379042167>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AQVmtQGvtfx14RVFCrKJsq6KaNd27hm2ks5tlmmCgaJpZM4TI8Fx>
> .
>

Minisolarclown commented:
Oh, okay. Like I said, it doesn't get used during matches. I use it to check the distance per pulse values on the encoder. I'll need it there to test a new encoder in a couple hours, but I can remove it once I'm done. Could I just replace it with getRawButtonReleased to make it only do it once when I need it to?

We did find something that may have caused the issue. [This](https://www.chiefdelphi.com/forums/showthread.php?t=163372) is a link that was sent to me in one of my Chief Delphi posts. In the post by Fletch1373, he says the issue was "solved" when they removed the camera. When we were troubleshooting on Monday, I did three things at the same time: switched how I was connected to the rio (via radio+ethernet), removed all camera code, and unplugged the camera. I do not know which of those three things got it working, but we didn't have the issue for the remaining half hour of unbag time.

JoshuaJacobson commented:
Get raw button released would be perfect.

That's an odd error, but I'd assume it is the unplugging of the camera that
did it.

On Thu, Apr 5, 2018, 15:09 Connor Williams <notifications@github.com> wrote:

> Oh, okay. Like I said, it doesn't get used during matches. I use it to
> check the distance per pulse values on the encoder. I'll need it there to
> test a new encoder in a couple hours, but I can remove it once I'm done.
> Could I just replace it with getRawButtonReleased to make it only do it
> once when I need it to?
>
> We did find something that may have caused the issue. This
> <https://www.chiefdelphi.com/forums/showthread.php?t=163372> is a link
> that was sent to me in one of my Chief Delphi posts. In the post by
> Fletch1373, he says the issue was "solved" when they removed the camera.
> When we were troubleshooting on Monday, I did three things at the same
> time: switched how I was connected to the rio (via radio+ethernet), removed
> all camera code, and unplugged the camera. I do not know which of those
> three things got it working, but we didn't have the issue for the remaining
> half hour of unbag time.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/CyberCoyotes/2018PowerUp/pull/1#issuecomment-379045068>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AQVmtWzBaAFkxK1OVWqk3QRec-yBgNLKks5tlmv1gaJpZM4TI8Fx>
> .
>

JoshuaJacobson commented:
I'll be in Alpena if you need any more help

On Thu, Apr 5, 2018, 15:18 Connor Williams <notifications@github.com> wrote:

> It is weird. There are a couple other teams that have the same issue. As
> far as I know, no one knows the real cause- only that unplugging the camera
> makes it work. My drivers said they do use it, and that it isn't a big deal
> to go without it. I brought an axis camera with me if they do end up
> needing the camera. I'm figuring that the way the axis camera communicates
> will bypass the weird issue.
>
> We don't know if it's our actual issue, but it's the biggest lead we have.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/CyberCoyotes/2018PowerUp/pull/1#issuecomment-379047548>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AQVmtb80-WcstGvBEElBD36CiDl8EKa6ks5tlm4dgaJpZM4TI8Fx>
> .
>
